x-pack/plugins/aiops/public/components/change_point_detection/change_point_detection_page.tsx

```diff
@@ -140,15 +200,16 @@ export const ChangePointDetectionPage: FC = () => {
       <EuiFlexGrid columns={annotations.length >= 2 ? 2 : 1} responsive gutterSize={'m'}>
         {annotations.map((v) => {
           return (
-            <EuiFlexItem key={v.group_field}>
+            <EuiFlexItem key={v.group.value}>
               <EuiPanel paddingSize="s" hasBorder hasShadow={false}>
                 <EuiFlexGroup justifyContent={'spaceBetween'} alignItems={'center'}>
                   <EuiFlexItem grow={false}>
                     <EuiFlexGroup alignItems={'center'} gutterSize={'s'}>
                       <EuiFlexItem grow={false}>
-                        <EuiTitle size="xxs">
-                          <h3>{v.group_field}</h3>
-                        </EuiTitle>
+                        <EuiDescriptionList
```

Comment:
Is there somewhere we can display the function and metric field? Currently there is no easy way to copy the metric field name.

Maybe to the left or right of the aggregation interval label? Or in the header next to the split field name / value - this will be repeated for every chart currently, but when we allow multiple series to be selected this info would be different.

<img width="773" alt="image" src="https://user-images.githubusercontent.com/7405507/217227254-e084e32a-ae59-4c3d-98bf-b780a10b2f5c.png">

Reply:
Placed function + metric field in the top right corner of the panel (af2cbdbe68939c811d566b39dd5524fdc7ba3ff9)

<img width="696" alt="image" src="https://user-images.githubusercontent.com/5236598/217253805-4bff4640-aa3e-4f49-b56d-c321843dda33.png">